app/src/main/java/org/simple/clinic/scheduleappointment/ScheduleAppointmentSheet.kt

```diff
@@ -17,7 +17,8 @@ import kotterknife.bindView
 import org.simple.clinic.R
```

Comment:
Reword this commit to something more descriptive? I don't understand what is being restricted and why.